LTI-JGG/LTI-JGG.md

```
@@ -0,0 +1,380 @@
+        DB["(PostgreSQL 16)"]
+        Storage["(Object Storage)"]
+        IA["(Asistente IA externo - opcional)"]
+    end
+
+    Web -->|HTTPS/JWT| API --> Core
+    Core --> DB
+    Core --> Storage
+    Core -.textos generados .-> IA
+```
+
+**Estrategia de despliegue y evolución**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)